egs2/dns_ins20/enh1/README.md

```diff
@@ -44,3 +44,27 @@
 
 Note: Here, the model is only trained on data without reverberation.
 Note: Here, the PESQ score is calculated based on https://github.com/vBaiCai/python-pesq.
+
+
+<!-- Generated by ./scripts/utils/show_enh_score.sh -->
+# RESULTS
+## Environments
+- date: `Fri Feb 25 05:12:24 EST 2022`
+- python version: `3.8.12 (default, Oct 12 2021, 13:49:34)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.6a1`
+- pytorch version: `pytorch 1.10.2+cu102`
+- Git hash: `21c02c8f578b9860e6bf38c86a0bd7cd0412c7f8`
+  - Commit date: `Sun Feb 6 15:37:51 2022 -0500`
+
+
+## enh_train_enh_conv_tasnet_raw
+
+- config: ./conf/tuning/train_enh_conv_tasnet.yaml
+- model: https://huggingface.co/muqiaoy/muqiaoy_dns_ins20_enh_train_enh_conv_tasnet_raw
+
+
+| dataset                           | STOI | SAR   | SDR   | SIR  |
```

Comment:
Could you remove the SIR column and add SI_SNR?

Reply:
Thanks for reminding. I have added the SI_SNR. Could you take a further look?